config/_shared.json

```diff
@@ -1,3 +1,4 @@
 {
-	"discover_blog_id": 53424024
+	"discover_blog_id": 53424024,
+	"project": "project"
```

Comment:
Actually I added another one called `project` inside `mc` subfolder, but it can be also the other way around :)